### ISSUE TITLE
Bug: lazy loading of packages does not find resources

### ISSUE THREAD

**Issue by clord** [grafana/plugin-tools#229]
### Which package(s) does this bug affect?

- [X] Create Plugin
- [ ] Sign Plugin

### Package versions

```shell
@grafana/create-plugin@1.2.1
```

### What happened?

When using react's lazy loading `const I = lazy(()=>import('path'))`, the file is not found. 

### What you expected to happen

I expect to get the right module when using import statements. 

As a hack attempt to make it work, I changed `.config/webpack/webpack.config.ts` to have a public path that points into the dist dir of our plugin ([Grafana Incident](https://github.com/grafana/incident/)) and found that lazy loading starts working, and some npm modules start using lazy loading as well. 

My change is below:

```js
  output: {
    clean: {
      keep: new RegExp(`.*?_(amd64|arm(64)?)(.exe)?`),
    },
    filename: '[name].js',
    library: {
      type: 'amd',
    },
    path: path.resolve(process.cwd(), DIST_DIR),
    publicPath: `public/plugins/${pluginJson.id}/`,
  },
````

### How to reproduce it (as minimally and precisely as possible)

use dynamic imports in some way, e.g., 

```js
const I = lazy(()=>import('path'));
...
function X() {return <I/>}
```

### Environment

```shell
this does not seem environment-specific.
```

### Additional context

I'm not sure my hack will work for production builds, will need some investigation. Thanks!

**Comment by clord** [comment]
@leventebalogh fyi. thoughts on this? would love to get this working.

**Comment by jackw** [comment]
@clord thanks for opening this issue. We certainly need to support dynamic chunk loading. Your proposed change should work but it will require testing across plugins to make sure it doesn't break anything else. 

In the meantime I would suggest you follow the [create-plugin documentation guide](https://grafana.github.io/plugin-tools/docs/advanced-configuration#extend-the-webpack-config) for customising the webpack config.

**Comment by clord** [comment]
Thanks @jackw

**Comment by jackw** [comment]
I'm closing this as it was fixed in #427 and #464
